packages/bautajs-fastify/README.md

```diff
@@ -44,7 +44,17 @@ By default the following plugins are included on the fastify instance
 
 ### Validation
 
-Fastify has a out-of-the-box request validation, therefore the builtin Bauta.js request validations is disabled on this plugin.
+### Request validation
+
+Fastify has a out-of-the-box request validation, since this validation takes place first, any error in the schema validation will be generated by fastify and not by `bautajs-core`. Therefore, even if you disable the validation of the request using the [bautajs-core methods](https://github.com/axa-group/bauta.js/blob/main/docs/validation.md#request-validation), the fastify validation will take place anyway.
```

Comment:
```suggestion
Fastify has an out-of-the-box request validation, since this validation takes place first, any error in the schema validation will be generated by fastify and not by `bautajs-core`. Therefore, even if you disable the validation of the request using the [bautajs-core methods](https://github.com/axa-group/bauta.js/blob/main/docs/validation.md#request-validation), the fastify validation will take place anyway.
```